rtbkit/common/plugin_table.h

```diff
@@ -0,0 +1,109 @@
+/* plugin_table.h
+   Flavio Moreira, 15 December 2014
+   Copyright (c) 2014 Datacratic.  All rights reserved.
+*/
+
+#pragma once
+
+#include <string>
+#include <unordered_map>
+#include <boost/any.hpp>
+#include "jml/arch/spinlock.h"
+
+namespace RTBKIT {
+
+struct PluginTable
```

Comment:
Looks good to me :)

On Wed, Jan 7, 2015 at 11:21 AM, oktal <notifications@github.com> wrote:

> In rtbkit/common/plugin_table.h
> <https://github.com/datacratic/rtbkit/pull/116#discussion-diff-22596041>:
>
> > @@ -0,0 +1,109 @@
> > +/* plugin_table.h
> > +   Flavio Moreira, 15 December 2014
> > +   Copyright (c) 2014 Datacratic.  All rights reserved.
> > +*/
> > +
> > +#pragma once
> > +
> > +#include <string>
> > +#include <unordered_map>
> > +#include <boost/any.hpp>
> > +#include "jml/arch/spinlock.h"
> > +
> > +namespace RTBKIT {
> > +
> > +struct PluginTable
>
> Ok so now you have one static instance of the PluginTable per Factory type
> ?
> I think I'm ok with that, @EricRobert <https://github.com/EricRobert> ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/datacratic/rtbkit/pull/116/files#r22596041>.
>